example/functions.pyi

```diff
@@ -1,3 +1,3 @@
 def double(x, /): ...
 def variadic(hello, /, *args, **kwargs): ...
-def with_kwargs(x, /, *, y=42.0): ...
+def with_kwargs(x, /, *, y=42): ...
```

Comment:
why this change? https://github.com/spiraldb/ziggy-pydust/blob/develop/example/functions.zig#L24

Reply:
I think the semantics of printing floats changed? under Zig 0.15.1, the following test program will print "42".

```zig
pub fn main() !void {
    // ignorable boilerplate
    var buf: [4096]u8 = undefined;
    var bw = std.fs.File.stdout().writer(&buf);
    const stdout = &bw.interface;

    try stdout.print("{d}", .{@as(f64, 42.0)});
    try bw.end();
}

const std = @import("std");
```

If the change is undesirable, I can make a change to `valueToStr`? https://github.com/spiraldb/ziggy-pydust/blob/develop/pydust/src/functions.zig#L544